### ISSUE TITLE
Wrong default "TEST_HOST" value (at least for macOS app target in Xcode 8)

### ISSUE THREAD

**Issue by maximkhatskevich** [jcampbell05/xcake#114]
If create a macOS app project with manually with Xcode, then unit tests target has "TEST_HOST" setting set like this:

``` Ruby
configuration.settings["TEST_HOST"] = "$(BUILT_PRODUCTS_DIR)/" + target.name + ".app/Contents/MacOS/" + target.name
```

If create project file with Xcake - it looks like this:

``` Ruby
configuration.settings["TEST_HOST"] = "$(BUILT_PRODUCTS_DIR)/" + target.name + ".app/" + target.name
```

Not sure if it is a macOS versus iOS, Xcode 7 versus Xcode 8 difference, or both. Need to investigate.

**Comment by jcampbell05** [comment]
Will look into this, I've been overhauling the build settings for our unit tests in 0.8

**Comment by jcampbell05** [comment]
Should be fixed in #153
